tests/transactional/host_config.pm

```diff
@@ -1,24 +1,37 @@
 # SUSE's openQA tests
 #
-# Copyright 2020-2023 SUSE LLC
+# Copyright 2020-2024 SUSE LLC
 # SPDX-License-Identifier: FSFAP
 #
 # Package: transactional-update
-# Summary: Host configuration operations (e.g. disable grub timeout,
-#              kernel params, etc)
+# Summary: Host configuration operations
+# * Configure virtio-terminal
+# * Configure bootloader (disable timeout, extra kernel parameters)
+# * Install SUSE CA certificates, if applicable
 #
 # Maintainer: QE-C team <qa-c@suse.de>
 
 use Mojo::Base qw(consoletest);
 use testapi;
 use transactional qw(process_reboot);
 use bootloader_setup qw(change_grub_config);
-use utils qw(ensure_ca_certificates_suse_installed zypper_call);
+use utils qw(ensure_ca_certificates_suse_installed zypper_call ensure_serialdev_permissions);
 use version_utils qw(is_bootloader_grub2 is_bootloader_sdboot);
+use serial_terminal qw(prepare_serial_console);
 
 sub run {
     select_console 'root-console';
 
+    # Ensure testuser (UID 1000) is present for the user-virtio-terminal
+    # this needs to happen before ensure_serialdev_permissions
+    if (script_run("getent passwd 1000") != 0) {
```

Comment:
This works, but I would use just `id 1000`.

Reply:
amended.